bibangamba/react-fast-food-fast
# #163720111 Setup redux

#### What does this PR do?
This Pull Request sets up the folder/file structure for `Redux` and installs it too

#### Description of tasks to be completed?
* install redux, redux-thunk, and redux-dev-tools-extension
* create folder structure for redux

#### How can this be manually tested?
* Install the `Redux Devtools` chrome extension
* Clone the repo
* Check out the `ch-redux-setup-163720126` branch
* run `yarn install`
* run `yarn start` to start the development server
* confirm that the `Redux Devtools` extension is activated meaning the application is using redux

#### What are the relevant pivotal tracker stories?
 #163720126
